tests/pds/pds_restore_test.go

```diff
@@ -157,6 +157,143 @@ var _ = Describe("{PerformRestoreToSameCluster}", func() {
 	})
 })
 
+var _ = Describe("{PerformRestoreToDifferentCluster}", func() {
+	bkpTargetName = bkpTargetName + pdsbkp.RandString(4)
```

Comment:
Instead of adding "pdsbkp.RandString(4)" in the testcase, cant we have it where "bckpTargetName" is defined. It removed the dependency on the coder to add the random generator.

Reply:
let it be like this for now, this name should be changed across all the testcases